tests/test_sparkkernelbase.py

```diff
@@ -65,16 +65,21 @@ def test_get_config():
     conf.load()
 
 
+# We check that the kernel goes into error state when configuration
+# is not set properly
 @with_setup(_setup, _teardown)
 def test_get_config_not_set():
     conf.override_all({})
-    try:
-        kernel._get_configuration()
+    kernel._get_configuration()
+    assert kernel._fatal_error
+    conf.load()
 
-        # Above should have thrown because env var not set
-        assert False
-    except ValueError:
-        assert send_error_mock.call_count == 1
```

Comment:
The intent of this unit test was to test that user got an error sent to him (displayed) immediately when configuration is not there. Does that work?